subprojects/file-watching/src/main/java/org/gradle/internal/watch/vfs/impl/WatchingVirtualFileSystem.java

```diff
@@ -199,6 +207,10 @@ public FileSystemWatchingStatistics getStatistics() {
                             return fileSystemWatchingStatistics;
                         }
                     });
+                    if (verboseLogging) {
```

Comment:
I think we should also log the number of watched hierarchies and whether or not watching has been stopped during the build.

For the build started logging, we also should log the number of watched hierarchies and whether watching has been started.

Reply:
I'll add a separate PR for this on top of #14342.